objects/CameraShot.js

```diff
@@ -1,25 +1,33 @@
+import Instrument from "./Instrument";
+
 /*
  * Class for storing camerashots
  */
 class CameraShot {
-    constructor(beginCount, endCount, name, description, colliding, instance, presetId) {
+    constructor(beginCount, endCount, name, description, colliding,
+                instance, presetId, instruments) {
         this.beginCount = beginCount;
         this.endCount = endCount;
         this.name = name;
         this.description = description;
         this.colliding = colliding;
         this.instance = instance;
         this.presetId = presetId;
+        this.instruments = instruments;
     }
 
     static fromXML(XMLObject) {
+        const instruments = XMLObject.instruments.map(
+            instrument => Instrument.fromXML(instrument));
         return new CameraShot(XMLObject.beginCount[0],
             XMLObject.endCount[0], XMLObject.name[0],
             XMLObject.description[0], XMLObject.colliding[0],
-            XMLObject.instance[0], XMLObject.presetId[0]);
+            XMLObject.instance[0], XMLObject.presetId[0],
```

Comment:
Just a question, do we still need the instance id? I can't come up with the reason we still need this.